packages/nouns-contracts/contracts/governance/ExcessETHBurner.sol

```diff
@@ -125,14 +126,16 @@ contract ExcessETHBurner is Ownable {
         // Make sure this is a valid noun id. This will revert if this id doesn't exist
         auction.nouns().ownerOf(currentNounId);
 
-        if (currentNounId < nextBurnNounID) revert NotTimeToBurnYet();
+        uint128 nextBurnNounID_ = nextBurnNounID;
+        if (currentNounId < nextBurnNounID_) revert NotTimeToBurnYet();
 
         amount = excessETH();
         if (amount == 0) revert NoExcessToBurn();
 
         IExecutorV3(owner()).burnExcessETH(amount);
 
-        nextBurnNounID += minNewNounsBetweenBurns;
+        nextBurnNounID = nextBurnNounID_ + minNewNounsBetweenBurns;
+        emit Burn(amount, nextBurnNounID_, nextBurnNounID);
```

Comment:
names of variables here feel confusing, and there's a gas nit here of reading `nextBurnNounID` from storage right after calculating its value in this function.

maybe store the new value in a local variable like `newNextBurnNounID`?
maybe change `nextBurnNounID_` to `nextBurnNounIDCache` to further signal it's caching the previous value?

Reply:
fixed: ea002b57cb3df093d5fc593678896977c46a021c